<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AssRules</class>
 <widget class="QWidget" name="AssRules">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>854</width>
    <height>591</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Association Rules</string>
  </property>
  <widget class="QGroupBox" name="groupBox_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>881</width>
     <height>571</height>
    </rect>
   </property>
   <property name="title">
    <string>Rules</string>
   </property>
   <widget class="QTableWidget" name="rulesTableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>811</width>
      <height>391</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Antecedent</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string/>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Consequent</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Support (%)</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Confidence (%)</string>
     </property>
    </column>
   </widget>
   <widget class="QGroupBox" name="groupBox_9">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>271</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Antecedent</string>
    </property>
    <widget class="QPushButton" name="btnFilterAnt">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>50</y>
       <width>71</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Filter</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>20</y>
       <width>241</width>
       <height>22</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Contains</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txtFilterContainAnt"/>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_10">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>30</y>
      <width>271</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Consequent</string>
    </property>
    <widget class="QPushButton" name="btnFilterConseq">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>50</y>
       <width>71</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Filter</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget_2">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>20</y>
       <width>241</width>
       <height>22</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>Contains</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txtFilterContainConsq"/>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_8">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>30</y>
      <width>251</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Information</string>
    </property>
    <widget class="QWidget" name="layoutWidget_3">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>22</y>
       <width>201</width>
       <height>74</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>From Date:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>To Date:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lblMinSupportFilter_2">
          <property name="text">
           <string>Min. Support (%):</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lblMinSupportFilter_3">
          <property name="text">
           <string>Min. Confidence (%):</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QLabel" name="lblFomDateFilter">
          <property name="text">
           <string>12</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lblToDateFilter">
          <property name="text">
           <string>12</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lblMinSupportFilter">
          <property name="text">
           <string>12</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lblMinConfFilter">
          <property name="text">
           <string>12</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
